src/vkvg_context.c

```diff
@@ -520,6 +523,13 @@ void vkvg_move_to (VkvgContext ctx, float x, float y)
 	_finish_path(ctx);
 	_add_point (ctx, x, y);
 }
+
+bool vkvg_has_current_point (VkvgContext ctx) {
+    if (ctx->status)
+        return false;
+    return true;
```

Comment:
Are you sure the context has always a current point?? :-)

Reply:
Maybe not.

Reply:
there are already private methods that check the current point...throw an eye on them to have an idea of the principle.

Reply:
Would it suffice to check `ctx->pointCount >= 0`?

Reply:
Wait is 👀 a yes or no haha